docs/interfaces/data_product_hub/discovery.md

```diff
@@ -28,13 +28,13 @@ A simple yet powerful way to find Data Products is provided by typing the name,
 
 ### **Filters**
 
-Multiple filter options further help narrow the search using parameters such as **Tier**, **Domain**, and **Owner**.
+Multiple filter options further help narrow the search using parameters such as [Tier](/interfaces/data_product_hub/core_concepts/#tier), [Domain](/interfaces/data_product_hub/core_concepts/#domain), and [Owner](/products/data_product/core_concepts/#data-product-owner).
 
-- **Tier:** Describes how the data product is organized, whether it follows a medallion architecture, is source-aligned, or is structured around specific entities or consumer needs.
+- **Tier:** Describes how the Data Product is organized, whether it follows a medallion architecture, is source-aligned, or is structured around specific entities or consumer needs.
 
-- **Domain:** Specifies the business domain to which the data product belongs. Multiple domains can be selected at once to filter for the desired Data Product.
+- **Domain:** Specifies the business domain to which the Data Product belongs. Multiple domains can be selected at once to filter for the desired Data Product.
 
-- **Owner:** Filters can be applied by owner, allowing users to direct questions to the appropriate person or team. The owner's DataOS User ID is used to list the various owners in the Owner dropdown, and only a single owner can be selected at a time.
+- **Owner:** Filters can be applied by owner, allowing users to direct questions to the appropriate person or team. The owner's DataOS User ID is used to list the various owners in the owner dropdown, and only a single owner can be selected at a time.
```

Comment:
Why have you small cased owner, just a question?

Reply:
fixed